daprdocs/content/en/dotnet-sdk-docs/_index.md

```diff
@@ -9,6 +9,24 @@ no_list: true
 
 Dapr offers a variety of packages to help with the development of .NET applications. Using them you can create .NET clients, servers, and virtual actors with Dapr.
 
+## Installation
+
+To get started with the .NET SDK, install the Dapr .NET SDK package:
+
+```sh
+dotnet install dapr
```

Comment:
Where did this command come from/are we just trying to use the packages or are we hoping for more here?

Reply:
I used the wrong command - my bad. Ok if I use this one instead?:

> `dotnet add package Dapr.Client --version 1.9.0`

Reply:
@hhunter-ms  - Adding the version means we'll have to update the doc with each release. Can we keep it versionless and say that that command will add the latest instead?

Reply:
@halspang that works for me, I'll update